src/assets/images/icons/arrow-primary.inline.svg

```diff
@@ -1,13 +1,13 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="19" height="16" viewBox="0 0 19 16">
-    <g fill="none" fill-rule="evenodd" stroke-linecap="round">
-        <g stroke="#FFF" stroke-width="2.4">
-            <g>
-                <g>
-                    <path d="M9.6 0L16 6.4" transform="translate(-507 -726) translate(280 688) translate(228 40)"/>
-                    <path d="M10.027 6.4L16.427 12.8" transform="translate(-507 -726) translate(280 688) translate(228 40) matrix(-1 0 0 1 26.027 0)"/>
-                    <path d="M0.427 6.4L15.36 6.4" transform="translate(-507 -726) translate(280 688) translate(228 40)"/>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can remove the metadata. Also from the svg tag, and the title tag. https://jakearchibald.github.io/svgomg/ is a good svg minifier

Reply:
Yeah, I think that we need to talk with design in the future to agree on a standard way of exporting svgs (for example, just 1 decimal in the paths, avoid metadata etc)

Reply:
SVG cleaned up